src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/BeneficialOwnerGovernmentOrPublicAuthorityDao.java

```diff
@@ -44,6 +44,9 @@ public class BeneficialOwnerGovernmentOrPublicAuthorityDao {
     @Field("ceased_date")
     private LocalDate ceasedDate;
 
+    @Field("ch_reference")
```

Comment:
ch_reference looks incorrect here?

Reply:
Okay, looks the same in the other files. So ch_reference is chips reference - but is that also the ID in self.links?

Reply:
It's part (last component) of the ID in self.links